plugin/Record/RecordManagerReadListViewEstate.php

```diff
@@ -305,17 +331,21 @@ private function readFieldconfigByListviewId($listviewId)
 	 *
 	 */
 
-	private function readContactDataByListviewId($listviewId)
+	public function getContactDataByListviewId(int $listviewId) : array
 	{
 		$prefix = $this->getTablePrefix();
 		$pWpDb = $this->getWpdb();
 
-		$sqlFields = "SELECT `fieldname`
-				FROM {$prefix}oo_plugin_listview_contactperson
-				WHERE `listview_id` = ".esc_sql($listviewId)."
-				ORDER BY `order` ASC";
+		$sql = $pWpDb->prepare(
+			"SELECT *
```

Comment:
Auch hier sollte doch nur `fieldname `selektiert werden, oder? 
```suggestion
			"SELECT `fieldname`
```